### PR TITLE
Fix for test-data make target

### PR DESCRIPTION
**Pull request by mkraft**
#### Summary

Fixes `make test-data` to work with mmctl now that the cli has removed the `config` and `sampledata` commands.

#### Ticket Link

n/a

#### Release Note

```release-note
NONE
```

### CODE REVIEW

**Review by cpoile — APPROVED** [review]
Thank you! I hope the sleep works, I'm a little concerned about that.

**Comment by mkraft** [comment]
> Thank you! I hope the sleep works, I'm a little concerned about that.

Yeah agreed. 

> Thank you! I hope the sleep works, I'm a little concerned about that.

You're right. I've improved it now, please have a re-review.

**Review by cpoile — APPROVED** [review]
Awesome!

File: Makefile [hunk 1]
```diff
@@ -425,9 +425,15 @@ cover: ## Runs the golang coverage tool. You must run the unit tests first.
...
 
-test-data: start-docker ## Add test data to the local instance.
-	$(GO) run $(GOFLAGS) -ldflags '$(LDFLAGS)' $(PLATFORM_FILES) config set TeamSettings.MaxUsersPerTeam 100
-	$(GO) run $(GOFLAGS) -ldflags '$(LDFLAGS)' $(PLATFORM_FILES) sampledata -w 4 -u 60
+test-data: run-server ## Add test data to the local instance.
+	@if ! ./scripts/wait-for-system-start.sh; then \
+		make stop; \
+	fi
+
+	@echo ServiceSettings.EnableLocalMode must be set to true.
```
Comment (streamer45) [inline Makefile:433]: Wondering if we should automate this using `jq` on `config.json` and force a server restart instead to make sure it gets applied. After all, this is a dev environment we are talking about.

Comment (mkraft) [reply Makefile:433]: I was debating that and thought it seemed like a bit of an overreach and was preferring to iterate on that addition if we think it's necessary in the future. Thoughts?

Comment (streamer45) [reply Makefile:433]: Fair enough :+1:

**Review by streamer45 — APPROVED** [review]

**Pull request merged**